src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -99,12 +124,21 @@ const CityColumnChart = (props) => {
     xAxis: {
       type: 'category',
       categories: data.map(city => city.name),
+      selectedCategory,
       labels: {
         rotation: -45,
         style: {
           fontSize: '10px',
         },
+        formatter() {
+          if (this.value === this.axis.userOptions.selectedCategory) {
```

Comment:
If this just passes in `selectedCategory` only to read it back out as `this.axis.userOptions.selectedCategory` why not just skip the middle man? Especially since 'selectedCategory' isn't a real property which makes it a bit confusing (I went and lookup up the API to see what it did).

Also I think the style is off, and shouldn't be `em`. So, all together:

```js
formatter() {
  // on the city page, highlight the city name bold if it's the current city
  if (mainCity && this.value === mainCity.name) {
    return `<span style="font-weight: 700; font-size: 12px">${this.value}</span>`;
  }

  return this.value;
},
```

Reply:
Yeah, this is simpler, thanks